examples/getting-started-session-based/02-session-based-XLNet-with-PyT.ipynb

```diff
@@ -1298,11 +1298,8 @@
   }
  ],
  "metadata": {
-  "interpreter": {
-   "hash": "7b543a88d374ac88bf8df97911b380f671b13649694a5b49eb21e60fd27eb479"
-  },
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "t4rec-22.12",
```

Comment:
does this change make any actual difference, or is it just metadata from you running the notebook?

Reply:
Metadata from running the tests locally.